packages/std/src/traits.rs

```diff
@@ -12,8 +12,8 @@ use crate::query::{
 };
 #[cfg(feature = "staking")]
 use crate::query::{
-    AllDelegationsResponse, BondedDenomResponse, Delegation, DelegationResponse, FullDelegation,
-    StakingQuery, Validator, ValidatorsResponse,
+    AllDelegationsResponse, AllValidatorsResponse, BondedDenomResponse, Delegation,
```

Comment:
Properly feature gated here though

Reply:
This is probably the same story we had with the messages: the staking related query types are public as part of `QueryRequest` no matter what compile flags are used. Will pull this change out into a separate PR.

Reply:
Sounds good. Yes, same story